### DISCUSSION TITLE
[FE] テーブルどうする問題

### DISCUSSION THREAD

**Discussion by Rinasham** [Hackathon5-TeamD/evolution-quest#46] [Q&A]
UserページとResultページのテーブル作成時にライブラリとか使うのか、それとも素でテーブル書くのか問題

**Comment by Terralien-jp** [comment]
for文イメージしてましたがリッチな方がいいですよねえｗ

**Comment by Rinasham** [comment]
一旦Semantic UIでトライしてあかんそうならその時考えますw
というか、Semantic UIが何か知らなさすぎてテーブルとか作れるってことをサイドバー調べてる時に初めて知りました(￣◇￣;)笑
謎なディスカッション立ててすいませんw

**Reply by Terralien-jp** [reply]
あーそういやテーブル作れますねｗｗ
